templates/gem/test/minitest_helper.rb

```diff
@@ -0,0 +1,44 @@
+=begin
+Copyright <%= copyright %>
+
+See the file LICENSE for copying permission.
+=end
+
+if
+  RUBY_VERSION == "2.0.0" # check Gemfile
```

Comment:
coverage is calculated only on this single ruby, needed gems are restricted only to this ruby https://github.com/jish/pre-commit/pull/233/files#diff-528248a33c4d64f323b7c6b622a98c54R13

on one hand it was needed for older rubies (I think redcarpet was not installing)
but it also helps to save workers time / test time for what it counts